cmake/api_macros.cmake

```diff
@@ -167,3 +167,20 @@ macro(OPENDDS_TARGET_SOURCES target)
 
   endforeach()
 endmacro()
+
+macro(ADD_DDS_IDL_FLAG flag)
```

Comment:
These names are in the global scope, so they should be prefixed with `OPENDDS_`.